### PR TITLE
[WIP] Create generic getter for CL version

### PR DESCRIPTION
**Pull request by squat**

### CODE REVIEW

**Comment by squat** [comment]
ok to test

**Comment by squat** [comment]
ok to test

**Comment by squat** [comment]
ok to test

File: modules/container_linux/main.tf [hunk 1]
```diff
@@ -0,0 +1,4 @@
+data "external" "version" {
+  count   = "${var.version == "latest" ? 1 : 0}"
+  program = ["sh", "-c", "curl http://${var.channel}.release.core-os.net/amd64-usr/ | grep '[0-9]\\+\\.[0-9]\\+\\.[0-9]\\+' | tail -n 1 | grep '[0-9]\\+\\.[0-9]\\+\\.[0-9]\\+' -o | tail -n1 | xargs -I {} echo '{\"version\": \"{}\"}'"]
```
Comment (lucab) [inline modules/container_linux/main.tf]: Is this equivalent to `https://$CHANNEL.release.core-os.net/amd64-usr/current/version.txt`? Also, please no plaintext HTTP.

Comment (squat) [reply modules/container_linux/main.tf]: thanks, I forgot that version.txt existed. This simplifies life considerably.

**Pull request closed without merging**